### ISSUE TITLE
Disable IPv6

### ISSUE THREAD

**Issue by intelliant01** [RechercheTech/mss#196]
With IPv6 enabled, the update process breaks for one as some systems are not able to resolve to repo.myscoolserver.com.

**Comment by intelliant01** [comment]
@cyberorg This breaks xrdp, hence Remmina due to this problem - https://github.com/neutrinolabs/xrdp/issues/714
xrdp 0.9.5 is there on the server.

**Comment by intelliant01** [comment]
https://github.com/neutrinolabs/xrdp/issues/1888#issuecomment-843180149
So the options are -
Either compile on your own with --disable-ipv6 or upgrade platform to 20.04 to get the latest xrdp which has this fixed.
@cyberorg Thus compiled package needed for immediate fix rollout.

**Comment by intelliant01** [comment]
Have rolled back the IPv6 disabling fix. Will manage this situation as of now by trying to manually disable IPv6 just before attempting an upgrade or when we encounter the address resolution error.

```
sudo sysctl -w net.ipv6.conf.all.disable_ipv6=1
sudo sysctl -w net.ipv6.conf.default.disable_ipv6=1
sudo sysctl -w net.ipv6.conf.lo.disable_ipv6=1
```
AFAIR, this fix had not worked, yet will try again since there is no other immediate solution.

**Comment by intelliant01** [comment]
Trying custom install script - https://c-nergy.be/blog/?p=15733

**Comment by intelliant01** [comment]
Compiled and installed -
xrdp_0.9.18-1_amd64.deb
xorgxrdp_0.2.18-1_amd64.deb
along with a few other dependencies and installed with sound support enabled and login screen customised -

![Untitled](https://user-images.githubusercontent.com/2577060/160282269-a3756458-6fd0-42fa-ad42-9ed00f9ac8fb.png)

Details of steps undertaken are in the updated hotfix [20220217-disable-ipv6](https://github.com/RechercheTech/mss/blob/master/fixes/mss/20220217-disable-ipv6)

Installer updated.
